command.go

```diff
@@ -71,58 +79,112 @@ func (c *Command) Run(ctx context.Context, stdout, stderr io.Writer) error {
 
 func RunAll(ctx context.Context, maxProcs int64, commands []*Command) error {
 	throttle := semaphore.NewWeighted(maxProcs)
-	errors := make(chan error, len(commands))
-	hasError := false
+	failures := make(chan *Failure, len(commands))
 	leftpad := 0
 	for _, command := range commands {
 		label := command.Label()
 		if len(label) > leftpad {
 			leftpad = len(label)
 		}
 	}
+	if leftpad < 3 {
+		leftpad = 3
+	}
```

Comment:
we can probably just initialize the variable with 3 before the loop yeah?

```go
// Set minimum to 3, matching the anchor used for non-program logs.
leftpad := 3
```

Reply:
LOL
I saw the section and never actually read the code inside the for loop 😆 
Completely missed this part: 

```go
		if len(label) > leftpad {
			leftpad = len(label)
		}
```